trywhatyoubuildDEVMODE-beg.adoc

```diff
@@ -0,0 +1,11 @@
+=== Try what you'll build
+
+The `finish` directory in the root of this guide contains the finished inventory application. Give it a try before you proceed.
```

Comment:
Actually this beginning sentence is a bit difficult to be common. Most (existing guides) have their own description about the application. Thus not much guide is really using this common adoc. IMO, it's still useful, but make it more generic. Couple things for this adoc: 

- [ ] Remove the application name, `inventory`
- [ ] Shorten the file name? ie, `twyb-intro.adoc`?  Don't think we need to use `devmode` in this file name, since the command is not dev mode.

Note, not sure when the `trywhatyoubuild-beg` was created.. it wasn't used in the template, so it's not the original. The `intro` version was used in the template; so used `intro` here.